doc/ss_logger.rst

```diff
@@ -0,0 +1,201 @@
+***************
+SmartSim Logger
```

Comment:
What do you think about removing SmartSim from this title.  Seeing it rendered in HTML, the hierarchy is SmartSim -> SmartSim Logger.  Feels redundant.

Reply:
Yes I will remove